classes/cf-input-settings-types.class.php

```diff
@@ -9,7 +9,18 @@
 */
 class cfs_input_block extends cf_input_block {
 	function __construct( $conf ) {
-		return cf_input_block::cf_input_block($conf);
+		return cf_input_block::cf_input_block( $conf );
+	}
+
+	/**
+	 * Added support for php 5.3-
```

Comment:
You should comment what the function does, not that you added support for PHP 5.3.